policies/templates/gcp_sql_instance_type_v1.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-sql-instance-type-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPSQLInstanceTypeConstraintV1
+      validation:
+        openAPIV3Schema:
+          properties:
+            mode:
+              type: string
+              enum: [allow, deny]
+              description: "String identifying the operational mode,
+              allow or deny. In the allow mode, only Cloud SQL instances
+              with an instance type in the instance_types list will be allowed (the other ones will
+              raise a violation. In the deny mode, the opposite is true. The defaults mode is deny."
```

Comment:
```suggestion
              raise a violation. In the deny mode, the opposite is true). The default mode is deny."
```

Reply:
fixed

Reply:
Looks like this is still here. Please check your commits aren't getting overwritten.

Reply:
I fixed the typo and added the missing ), just at the end of the previous sentence. I think it looks good now.